### PR TITLE
Tree model sort

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Reopening of #721 (I want to have it for next release!). I cherry-picked @ColonelThirtyTwo's commit so his ownership remains.

cc @sdroege @EPashkin

### CODE REVIEW

**Comment by EPashkin** [comment]
Need use `self.as_ref().to_glib_xxx` and `.unsafe_cast()`

**Comment by GuillaumeGomez** [comment]
And to remove an import as well.

File: Gir.toml [hunk 1]
```diff
@@ -1692,6 +1692,10 @@ status = "generate"
     # Replaced by TreeModelFilter::new
     ignore = true
     [[object.function]]
+    name = " sort_new_with_model"
+    # Replaced by TreeModelSort::new
+    ignore = true
```
[the remark this answers is not in the document]
Comment (sdroege) [reply Gir.toml]: Link it from inside the .toml file I mean

**Comment by sdroege** [comment]
:+1:

**Pull request merged**